tier1/checklist.md

```diff
@@ -0,0 +1,368 @@
+| Project Description | 1-3 sentences short description of the project that can be used as a 'one-liner' to describe the repo. A best practice is using this same language as the official 'description' on GitHUb repo landing page.                                                                                  |              |
+| About the Project   | Longer-form description of the project. It can include history, background, details,problem statements,links to design documents or other supporting materials,or any other information/context that a user or contributor might be interested in.                                             |              |
+| Core Team           | This information helps with succession planning and provenance for security compliances and remediation. It helps future users and contributors understand where the code originated.                                                                                                          |              |
+| Policies            | This section is to explicitly link to Federal policies and guidelines that are required or rocommended for Federal projects to comply with, such as Accessibility (508) Interoperability, Anti-deficiency, Security, Licensing, and other policies that can vary between agencies and domains. |              |
+| Public Domain       | A best practice is to list the LICENSE under which a project is released at the bottom of the README. In most cases for Federal repos, we default to Creative Commons Zero 1.0 International(world-wide public domain.)                                                                        |              |
+
+
+- [ ] **License**
+
+License of your project, whether public domain (CC0) or other OSI-approved License. Using 'vanilla' license text will allow for GitHub to auto-label the license information on the repository landing page.
+
+- [ ] **CONTRIBUTING.md**
+Provide guidance on how users can run your project and make contributions to it
+
+| **Section**           | **Description**                                                                                                                                                                                                                                                                                     | **Included** |
+|-----------------------|-----------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------|--------------|
+| Getting started       | Included installations steps, prerequisites for installation, and instructions for working with the source code                                                                                                                                                                                     |              |
```

Comment:
Make sure all headers match. Currently, "Getting started" should be "Getting Started"